force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -190,11 +219,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The result of the adjusted arithmetic expression.
      */
     public static Integer adjustOrderOfOperations1(Integer a, Integer b, Integer c) {
-        // Add parentheses around the addition operation so that it is performed before multiplication
-        Integer result = a + b * c;
-
+        // Add parentheses around the addition operation so that it is performed before multiplication        
+        Integer result = (a + b) * c;
+        System.debug('Value of a : ' + a + ' and b : ' + b + ' and c : ' + c + ' and Result = ' + result);
         // Return the result
-        return null; // Replace null with the variable you used to store the result
+        return result; // Replace null with the variable you used to store the result
```

Comment:
good work. A good habit to get in to is delete your system.debug statements. You don't want those going to Production